man/mkbundle.1

```diff
@@ -41,9 +41,9 @@ targets, and the "--cross" argument to specify the target, like this:
 	$ mkbundle --local-targets	
 	Available targets:
 		default	- Current System Mono
-		4.4.0-macosx-x86
-		4.4.0-debian-8-arm64
-	$ mkbundle --cross 4.4.0-debian-8-arm64 hello.exe -o hello-debian
+		4.4.1-macosx-x86
+		4.4.1-debian-8-arm64
+	$ mkbundle --cross 4.4.1-debian-8-arm64 hello.exe -o hello-debian
 .fi
 .PP
 The above will bundle your native library into hello-debian for
```

Comment:
I'd drop this change since this is an invalid target (see https://download.mono-project.com/runtimes/raw/target-list.txt and mcs/tools/mkbundle/mkbundle.cs). It's just an example anyway, and the valid target list can be retrieved via mkbundle itself.

And I forgot that the need for the winconfig.h bump was removed, so the configure.ac change is all we need.